cmd/cerca/cerca.go

```diff
@@ -27,24 +27,63 @@ func readAllowlist(location string) []string {
 	return processed
 }
 
-func complain(msg string) {
-	fmt.Printf("cerca: %s\n", msg)
+func inform(msg string, args ...interface{}) {
+	if len(args) > 0 {
+		fmt.Printf("%s\n", fmt.Sprintf(msg, args...))
+	} else {
+		fmt.Printf("%s\n", msg)
+	}
+}
+
+func complain(msg string, args ...interface{}) {
+	if len(args) > 0 {
+		inform(msg, args)
+	} else {
+		inform(msg)
+	}
 	os.Exit(0)
 }
 
-func main() {
+func cerca() {
 	// TODO (2022-01-10): somehow continually update veri sites by scraping merveilles webring sites || webring domain
 	var allowlistLocation string
 	var sessionKey string
 	var configPath string
 	var dataDir string
 	var dev bool
+
 	flag.BoolVar(&dev, "dev", false, "trigger development mode")
 	flag.StringVar(&allowlistLocation, "allowlist", "", "domains which can be used to read verification codes from during registration")
 	flag.StringVar(&sessionKey, "authkey", "", "session cookies authentication key")
 	flag.StringVar(&configPath, "config", "cerca.toml", "config and settings file containing cerca's customizations")
 	flag.StringVar(&dataDir, "data", "./data", "directory where cerca will dump its database")
+
```

Comment:
can we drop the first-line `NAME:` from each help output?

Reply:
thinking a bit more, maybe a util function in `cmd/cerca/` would be a good addition and reduce string repetition? 

i also think it is important to not lose concrete example invocations for each commands in this refactor. contrast the following two for the migration tool:

```
usage
	migration-tool --list 
	migration-tool --migration <name of migration>
```

is a lot more useful for orienting as a confused user / infrequent maintainer than

```
  cerca migrate [command] [options]
```

---
help util sketch

```golang
func createHelpString(commandName, usageOptions, exampleUsage string) string {
  return fmt.Sprintf(`%s
  USAGE:
  \t%s
  \t%s
  GLOBAL OPTIONS:
  \t-help
  \t\tshow help
  OPTIONS:`, commandName, usageOptions, exampleUsage)
}
```

`exampleUsage` could cover the expanded `COMMANDS: ` section for the main command

Reply:
@cblgh nice yep, can do. what do you mean by `usageOptions` in this example? the options are below in `OPTIONS: ...`? do you mean a `cerca [command] [options]` thing or 2 concrete examples?

Reply:
@decentral1se idk lol i think i initially was just putting it in there as a placeholder for your `cerca [command] [options]` string, with `exampleUsage` representing the examples proper. 

could totally skip the `[command] [options]` line though, and simply stick to outputting the name, the concrete examples (or in the case of the main `cerca` cmd, the `COMMANDS` listing), and finishing with section `OPTIONS` with `printDefaults` like you have atm

wdyt?

Reply:
looks a bit out of place to include the name when we just ran the command and the name is in the terminal. the rest sounds good! so here's what i came up with: https://github.com/cblgh/cerca/pull/93/commits/7151cb5e910cb885534d8da5b7c918ed6c95202c

Reply:
oh i like it!

> looks a bit out of place to include the name on a newline when we just ran the command and the name is in the terminal

i think i can live with this injury to my aesthetic sensibilities X)

thanks!